### DISCUSSION TITLE
Custom Params doc has code that doesn't work with Python 3.11?

### DISCUSSION THREAD

**Discussion by bjornhansen** [uriyyo/fastapi-pagination#793] [Q&A] [answered]
When I paste the example code from the documentation (Advanced User Guide --> Custom Params) into VSCode, Pylance is giving the error: `Variable not allowed in type expression` on line 25.

Am I doing something wrong? Is this a Python version thing? Or is this code example incorrect?

As of this writing, the example code is:
```py
from typing import List

from fastapi import FastAPI
from pydantic import BaseModel, EmailStr, Field

from fastapi_pagination import Page, add_pagination, paginate

Page = Page.with_custom_options(
    size=Field(100, ge=1, le=500),
)

app = FastAPI()
add_pagination(app)


class UserOut(BaseModel):
    name: str
    email: EmailStr


users: List[UserOut] = []


@app.get("/users")
def get_users() -> Page[UserOut]:
    return paginate(users)

```

**Comment by uriyyo** [comment, marked as answer]
@bjornhansen It's VS issue cause `with_custom_options` returns a new type and VS can't resolve it.

**Reply by bjornhansen** [reply]
Thanks for the insight!
